### PR TITLE
Expose WorkerType explicitly

### PR DESCRIPTION
**Pull request by keepingitneil**

### CODE REVIEW

**Review by theomonnom — APPROVED** [review]
tests are failing since last PR, will take a look later

File: livekit-agents/livekit/agents/__init__.py [hunk 1]
```diff
@@ -16,13 +16,13 @@
...
 from .plugin import Plugin
 from .version import __version__
-from .worker import Worker, WorkerOptions
+from .worker import Worker, WorkerOptions, WorkerType
 
 __all__ = [
     "__version__",
     "Worker",
     "WorkerOptions",
-    "WorkerPermissions",
```
Comment (theomonnom) [inline livekit-agents/livekit/agents/__init__.py:25]: should we remove?

Comment (keepingitneil) [reply livekit-agents/livekit/agents/__init__.py:25]: Didn't exist in the imports so figured it wasn't meant to be there

File: livekit-agents/livekit/agents/ipc/supervised_proc.py [hunk 2]
```diff
@@ -22,7 +22,7 @@ class LogQueueListener:
     _sentinel = None
 
     def __init__(
-        self, queue: mp.Queue, prepare_fnc: Callable[[logging.LogRecord], None]
+        self, queue: mp.Queue[Any], prepare_fnc: Callable[[logging.LogRecord], None]
```
Comment (theomonnom) [inline livekit-agents/livekit/agents/ipc/supervised_proc.py:25]: mp.Queue isn't generic

Comment (theomonnom) [reply livekit-agents/livekit/agents/ipc/supervised_proc.py:25]: [Any] shouldn't be necessary

File: livekit-agents/livekit/agents/__init__.py (hunk 1 above)
Comment (theomonnom) [reply livekit-agents/livekit/agents/__init__.py:25]: dz added it here https://github.com/livekit/agents/pull/625

Comment (theomonnom) [reply livekit-agents/livekit/agents/__init__.py:25]: I guess it needs to be imported here https://github.com/livekit/agents/blob/c12589d9cd782ffc28802694ca0b11ffc7c2d0cc/livekit-agents/livekit/agents/worker.py#L83

**Pull request merged**